content/en/docs/setup/platform-setup/openshift/index.md

```diff
@@ -13,10 +13,10 @@ test: no
 
 Follow these instructions to prepare an OpenShift cluster for Istio.
 
-Install Istio using the OpenShift profile:
+Install Istio using the OpenShift platform profile:
 
 {{< text bash >}}
-$ istioctl install --set profile=openshift
+$ istioctl install --set values.global.platform=openshift
```

Comment:
@bleggett, this command does not enable CNI which is required for OpenShift. Please see...

```
$: istioctl install --set values.global.platform=openshift
        |\          
        | \         
        |  \        
        |   \       
      /||    \      
     / ||     \     
    /  ||      \    
   /   ||       \   
  /    ||        \  
 /     ||         \ 
/______||__________\
____________________
  \__       _____/  
     \_____/        

This will install the Istio 1.24.0 profile "default" into the cluster. Proceed? (y/N) y
✔ Istio core installed ⛵️                                                                                                                                                                     
✔ Istiod installed 🧠                                                                                                                                                                         
✔ Ingress gateways installed 🛬                                                                                                                                                               
✔ Installation complete 

$: k get pods -n kube-system
NAME                       READY   STATUS    RESTARTS   AGE
coredns-58696f69f5-ndwkq   1/1     Running   0          98d
```

OTOH, if we use the openshift profile...

```
$: istioctl install --set profile=openshift
        |\          
        | \         
        |  \        
        |   \       
      /||    \      
     / ||     \     
    /  ||      \    
   /   ||       \   
  /    ||        \  
 /     ||         \ 
/______||__________\
____________________
  \__       _____/  
     \_____/        

This will install the Istio 1.24.0 profile "openshift" into the cluster. Proceed? (y/N) y
✔ Istio core installed ⛵️                                                                                                                                                                     
✔ Istiod installed 🧠                                                                                                                                                                         
✔ Ingress gateways installed 🛬                                                                                                                                                               
✔ CNI installed 🪢                                                                                                                                                                            
✔ Installation complete    

$: k get pods -n kube-system
NAME                       READY   STATUS    RESTARTS   AGE
coredns-58696f69f5-ndwkq   1/1     Running   0          98d
istio-cni-node-4s7vj       1/1     Running   0          36s
istio-cni-node-78jhz       1/1     Running   0          36s
istio-cni-node-9klq6       1/1     Running   0          36s
istio-cni-node-lff9v       1/1     Running   0          36s
istio-cni-node-wkdm7       1/1     Running   0          36s
istio-cni-node-xpbnz       1/1     Running   0          36s
$
```

Reply:
We'll have to modify the command as follows

```
istioctl install --set components.cni.enabled=true --set components.cni.namespace=kube-system --set values.global.platform=openshift
```